src/media-popup.js

```diff
@@ -54,4 +71,4 @@ ipcRenderer.on('load-media', (event, mediaUrl, mediaType) => {
     video.autoplay = true;
     video.playbackRate = 1;
   }
-});
+});
```

Comment:
Interesting, is there a reason for that, or is that just because that's how you style your documents?